tools/release/bump_version/versions.json

```diff
@@ -1,12 +1,12 @@
 {
     "gateway": 
     {
-        "core" : "1.0.4"
+        "core" : "1.0.5"
     },
     "bindings" :
     {
         "dotnet" : "1.0.2",
-        "dotnetcore": "1.0.2",
+        "dotnetcore": "1.0.3",
         "java" : "1.1.0"
```

Comment:
We didn't touch any .java files, so we don't need to bump versions inside the JARs. We made some changes to C code like the module loader, but that is considered part of "core" for now...